code/game/machinery/cloning.dm

```diff
@@ -239,8 +236,16 @@
 	//Here let's calculate their health so the pod doesn't immediately eject them!!!
 	H.updatehealth()
 
-	clonemind.transfer_to(H)
-	to_chat(H, "<span class='notice'><b>Consciousness slowly creeps over you as your body regenerates.</b><br><i>So this is what cloning feels like?</i></span>")
+	if(grab_ghost_when == CLONER_FRESH_CLONE)
+		clonemind.transfer_to(H)
+		H.ckey = R.ckey
+		to_chat(H, "<span class='notice'><b>Consciousness slowly creeps over you \
+			as your body regenerates.</b><br><i>So this is what cloning \
+			feels like?</i></span>")

```

Comment:
Eugh. Escaped newlines don't belong in strings, since we have `{" document syntax "}` for multiline text.